src/test/java/com/contrastsecurity/maven/plugin/ContrastScanMojoTest.java

```diff
@@ -38,4 +54,60 @@ void it_generates_clickable_url(final String url) throws MalformedURLException {
         .isEqualTo(
             "https://app.contrastsecurity.com/Contrast/static/ng/index.html#/organization-id/scans/project-id/scans/scan-id");
   }
+
+  @Test
+  void it_prints_summary_to_console() {
+    // GIVEN the plugin is configured to output scan results to the console
+    mojo.setConsoleOutput(true);
+
+    // WHEN print summary to console
+    final ScanSummary summary =
+        ScanSummary.builder()
+            .id("summary-id")
+            .organizationId("organization-id")
+            .projectId("project-id")
+            .scanId("scan-id")
+            .createdDate(LocalDateTime.now())
+            .totalResults(10)
+            .totalNewResults(8)
+            .totalFixedResults(1)
+            .duration(0)
+            .build();
+    @SuppressWarnings("unchecked")
+    final Consumer<String> console = mock(Consumer.class);
+    mojo.writeSummaryToConsole(summary, console);
+
+    // THEN prints expected lines
+    final List<String> expected =
+        Arrays.asList("Scan completed", "New Results\t8", "Fixed Results\t1", "Total Results\t10");
```

Comment:
OK. fwiw I don't mind repetition in tests like this (unless it's making them really brittle), because it's sort of like the unit test equivalent of double entry accountng.